src/SUnit-Core/TestCase.class.st

```diff
@@ -10,7 +10,8 @@ Class {
 	#superclass : #TestAsserter,
 	#instVars : [
 		'testSelector',
-		'expectedFails'
+		'expectedFails',
+		'aTestEnvironment'
```

Comment:
Why the name begins with a?

Reply:
The name without the a is taken by another slot in a subclass.
I will certainly change the name later.

Reply:
It could be "testingEnvironment”